packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -167,11 +168,20 @@ const UsersProfileFn = ({terms, slug, classes}: {
             </Components.MetaInfo>
           </span>
         </Tooltip>
+
+        <Tooltip title={`${tagRevisionCount} wiki edit${tagRevisionCount === 1 ? '' : 's'}`}>
+          <span className={classes.userMetaInfo}>
+            <ChangeHistoryIcon className={classNames(classes.icon, classes.specificalz)}/>
+            <Components.MetaInfo>
```

Comment:
The other icons pass in a `title` prop here but as far as I could tell that is not being used in `MetaInfo`? Let me know if I'm missing something